src/collapse/bpftrace.rs

```diff
@@ -1,20 +1,21 @@
+//! Module containting the **bpftrace** implementation of [`Frontend`].
```

Comment:
I almost wonder if we should _not_ document the module at this point, because otherwise we might not realize that we need to expand the docs when we later add `#![deny(missing_docs)]` to the crate.

Reply:
👍